### DISCUSSION TITLE
İbax is Cardano killer?

### DISCUSSION THREAD

**Discussion by Seyitreyhan** [IBAX-io/go-ibax#477] [Q&A] [answered]
### Clear and concise description of the problem

Ibax metaverse and dex, which have established their own network on open lines, have defi web3 features Will Cardano finish the unified network structure?

### Suggested solution

It makes a difference if it is as cheap and fast as the Tron network

### Alternative

For this, the dec-block transfer needs to be transferred to the minnig part very quickly?

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Seyitreyhan** [comment]
Lest go minnig

**Comment by HalloKyle** [comment, marked as answer]
The IBAX Network is fast and cheap enough that the ecosystem token can be used as a gas fee and this feature is unique in the market.

**Comment by Seyitreyhan** [comment]
we will be in the top 3

**Comment by Seyitreyhan** [comment]
It would be a very different phase if our network fees and speed moved to the mining phase.

**Comment by impossible77777** [comment]
Great projekt
